src/components/Header/Header.tsx

```diff
@@ -132,6 +134,8 @@ export const Header = ({
    *
    * Add scroll and window resize event to adjust values.
    */
+  // FIXME
+  // eslint-disable-next-line @chanzuckerberg/edu-react/use-effect-deps-presence
```

Comment:
We should still audit all our useEffects sometime.

There are definitely some that legit don't need a deps array. But most of these I'm not so sure.